src/app/zap-templates/zcl/data-model/chip/chip-ota.xml

```diff
@@ -110,9 +110,9 @@ limitations under the License.
     </enum>
     <struct name="ProviderLocation">
         <cluster code="0x002a"/>
-        <item fieldId="0" name="FabricIndex" type="fabric_idx"/>
         <item fieldId="1" name="ProviderNodeID" type="node_id"/>
         <item fieldId="2" name="Endpoint" type="endpoint_no"/>
+        <item fieldId="0xFE" name="FabricIndex" type="fabric_idx"/>
```

Comment:
add a `<!-- -->` comment explaining why this is here even though spec says it should not. Link to the github issue that describes the problem and how we should approach the solution.

Reply:
See https://github.com/project-chip/connectedhomeip/issues/15836 and https://github.com/project-chip/connectedhomeip/issues/15835

This PR is basically doing https://github.com/project-chip/connectedhomeip/issues/15835 for the OTA Requestor cluster.